cognite/neat/rules/exporters/_rules2excel.py

```diff
@@ -241,25 +257,29 @@ def _create_metadata_sheet_user_rules(self, rules: Rules) -> dict[str, Any]:
         # Excel does not support timezone in datetime strings
         now_iso = datetime.now().replace(tzinfo=None).isoformat()
         is_info = isinstance(rules, InformationRules)
-        is_dms = not is_info
-        is_extension = self.new_model_id is not None
-        is_solution = is_extension and self.new_model_id != existing_model_id
+        is_dms = isinstance(rules, DMSRules)
```

Comment:
This whole function needs a rewrite, have captured it in https://cognitedata.atlassian.net/browse/NEAT-240